### PR TITLE
[Improve]Some code block optimizations

### PR DESCRIPTION
**Pull request by CalvinKirs**

### CODE REVIEW

File: seatunnel-apis/seatunnel-api-spark/src/main/java/org/apache/seatunnel/spark/SparkEnvironment.java [hunk 1]
```diff
@@ -47,8 +47,6 @@
 
     private boolean enableHive = false;
 
-    private JobMode jobMode;
-
```
Comment (ruanwenjun) [inline seatunnel-apis/seatunnel-api-spark/src/main/java/org/apache/seatunnel/spark/SparkEnvironment.java]: Suggested don't remove this field, this field used to defined what the job mode of this job(streaming, batch, sql...), this field is useful, although it is not used in spark now. 

Remove the setJobMode method in RuntimeEnv looks good to me, we don't need to use this method, the jobMode should be set at prepare method. I would like add `getJobMode` method in RuntimeEnv.

This part of refactoring may require a separate PR to complete.

**Review by ruanwenjun — DISMISSED** [review]
Good job, just add a comment. But it will not affect this PR to be merged.

File: seatunnel-apis/seatunnel-api-spark/src/main/java/org/apache/seatunnel/spark/SparkEnvironment.java (hunk 1 above)
Comment (CalvinKirs) [reply seatunnel-apis/seatunnel-api-spark/src/main/java/org/apache/seatunnel/spark/SparkEnvironment.java]: ok...I revert this,

**Review by wuchunfu — APPROVED** [review]
LGTM

**Comment by CalvinKirs** [comment]
> LGTM

thx

**Pull request merged**